docs/model_structure.rst

```diff
@@ -30,12 +31,24 @@ The filenames for the components are of the form carrier-tech.csv (e.g. :file:`e
 * **carrier** Energy sector according to carrier (e.g. solar, wind, biomass, coal, lignite, uranium, oil, gas, methane, hydro, waste, electricity, heat).
 * **tech** Specification of the technology (e.g. st, ocgt, ccgt, pv, onshore, offshore, ror, phs, extchp, bpchp, battery)
 
+TODO: Explain that other columns that follow describe the attributes of the components.
+
 Sequences
 ---------
 
 The filenames are of the form type_profile (e.g.
 :file:`wind-offshore_profile.csv`, :file:`electricity-demand_profile.csv`).
 
+TODO: explain the columns and their names within the sequences files
+
+The found timeseries are combined into a new set of CSV files, with one file per technology and
+``{region code}-{component}-profile`` as column names.
+They are stored in ::
+
+    results/{scenario name}/01_preprocessed/data/sequences/{technology}_profile.csv
+
+for the optimization step.
+
 Available components
 ====================
 
```

Comment:
In my opinion it would be clearer without the middle column, by stating the path once for all. I don't know if it's possible in an rst file to remove a column.